qt/python/mantidqt/project/encoderfactory.py

```diff
@@ -6,6 +6,7 @@
 # SPDX - License - Identifier: GPL - 3.0 +
 #  This file is part of the mantidqt package
 #
+from mantid.simpleapi import logger
```

Comment:
Could we import logger from `mantid.kernel` instead? The simpleapi requires the whole framework to be initialized if this is imported from somewhere else. If we just want components then we should try and get them from the lower-level modules.